build/Helix/runtests.cmd

```diff
@@ -0,0 +1,9 @@
+set
+robocopy %HELIX_CORRELATION_PAYLOAD% . /s
+dir /b /s
+te MUXControls.Test.dll MUXControlsTestApp.appx /enablewttlogging /unicodeOutput:false /testtimeout:0:05 %*
+type te.wtl
```

Comment:
Why do we need type here, troubleshooting?

Reply:
Yes. It is for debugging purposes.